index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Project</title>
+</head>
+<body>
+
+    <div class="template-1">
+
+        <header>
+            <!-- Toolbar Nav -->
+    <nav class="toolbar-container">
+        <!-- Toolbar Name -->
+        <div class="toolbar-name-container">
+            <p class="toolbar-name">Wahab</p>
```

Comment:
- [ OPTIONAL] This is your logo, It is best to have the logo in a link tag and make it navigate to the home page. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

Reply:
i implemented it.